doc/releases/changelog-0.27.0.md

```diff
@@ -4,145 +4,227 @@
 
 <h3>New features since last release</h3>
 
-<h4>(TODO: title) JAX JIT</h4>
+<h4>An all-new data module 💾</h4>
 
-<h4>(TODO: title) Qutrits</h4>
+* The `qml.data` module is now available, allowing users to download, load, and create quantum datasets.
+  [(#3156)](https://github.com/PennyLaneAI/pennylane/pull/3156)
 
-* The `qml.GellMann` qutrit observable, the ternary generalization of the Pauli observables, is now available.
-  ([#3035](https://github.com/PennyLaneAI/pennylane/pull/3035))
+  Datasets are hosted on Xanadu Cloud and can be downloaded by using `qml.data.load()`:
 
-  When using `qml.GellMann`, the `index` keyword argument determines which of the 8 Gell-Mann matrices is used.
+  ```pycon
+  >>> H2_datasets = qml.data.load(
+    data_name="qchem", molname="H2", basis="STO-3G", bondlength=1.1
+  )
+  >>> print(H2_datasets)
+  [<Dataset = description: qchem/H2/STO-3G/1.1, attributes: ['molecule', 'hamiltonian', ...]>]
+  >>> H2data = H2_datasets[0]
+  ```
 
-  ```python
-  dev = qml.device("default.qutrit", wires=2)
+  - Datasets available to be downloaded can be listed with `qml.data.list_datasets()`.
 
-  @qml.qnode(dev)
-  def circuit():
-      qml.TClock(wires=0)
-      qml.TShift(wires=1)
-      qml.TAdd(wires=[0, 1])
-      return qml.expval(qml.GellMann(wires=0, index=8) + qml.GellMann(wires=1, index=3))
-  ```
+  - To download or load only specific properties of a dataset, we can specify the desired properties in `qml.data.load` with the `attributes` keyword argument:
+
+    ```pycon
+    >>> H2_hamiltonian = qml.data.load(
+      data_name="qchem", molname="H2", basis="STO-3G", bondlength=1.1,
+      attributes=["molecule", "hamiltonian"]
+    )[0]
+    >>> H2_hamiltonian.hamiltonian
+    <Hamiltonian: terms=15, wires=[0, 1, 2, 3]>
+    ```
+
+    The available attributes can be found using `qml.data.list_attributes()`:
+
+  - To select data interactively, we can use `qml.data.load_interactive`:
 
   ```pycon
-  >>> circuit()
-  -0.42264973081037416
+  >>> qml.data.load_interactive()
```

Comment:
Might want to reduce the indentation of the print below by one step to not have the horizontal scroll.